go/arm/Dockerfile.tmpl

```diff
@@ -102,23 +115,23 @@ COPY rootfs /
 
 # Basic test
 RUN cd / \
-  && aarch64-linux-gnu-gcc helloWorld.c -o helloWorld \
-  && file helloWorld \
-  && readelf -h helloWorld \
-  && file helloWorld | cut -d "," -f 2 | grep -c 'ARM aarch64'\
-  && rm helloWorld.c helloWorld
+        && aarch64-linux-gnu-gcc helloWorld.c -o helloWorld \
```

Comment:
If you want to change the indentation of the project is fine, but Could you doit in a separate PR to see here the real changes only.